proposal.rst

```diff
@@ -102,48 +110,73 @@ Background Information
 similar programmes before?  If so, tell us about how it went, and any areas you
 would have liked more help with.**
 
-FILLME
+No, I haven't.
+
 
 **Please tell us about any previous experience you have with Xapian, or other
 systems for indexed text search.**
 
-FILLME
+My first encounter with Xapian was when I installed xapian to index search results on synaptic package manger.
+After I grew interest to take part in Xapian project for Gsoc, I started by building xapian and getting familiar with the code base.
+I started working on adding `two new range operators <https://github.com/xapian/xapian/pull/289/>`_ (OP_VALUE_LT and OP_VALUE_GT), then I built two new testcases for them, and once these
+changes commited, I will start on writing a documentation for them.
+I also added a `minor optimization <https://github.com/xapian/xapian/commit/3c56e5db5b8f3696fd8f311793c62921eb413ef8/>`_ for "OP_VALUE_GE".
 
 **Tell us about any previous experience with Free Software and Open Source
 other than Xapian.**
 
-FILLME
+There was a small project which is a `command line interface for spotify on Linux <https://github.com/pwittchen/spotify-cli-linux/>`_
+which is written in python3, and it uses a `python2 api <https://github.com/enricobacis/lyricwikia/>`_ to get songs lyrics, but the two
+didn't build for me due to combatibility issues, so I `adapted them and get them to fully work <https://github.com/BodaSadalla98/spotify-cli-linux/>`_.
+
+
+|
 
 **What other relevant prior experience do you have (courses taken at college,
 hobbies, holiday jobs, etc)?**
 
-FILLME
+I took many Software related Courses in college like:
+
+	* MIPS processor and its assembly.
+ 	* c, c++, OOP with c++. I did some college projects like `Equations-solver <https://github.com/BodaSadalla98/Equations-Solver/>`_.
+	* Data structures and algorithms.
+	I have dived a little bit deeper in DS and algorithms due to my interest in compettive programming.
 
 **What development platforms, tools and methods do you prefer to use?**
 
-FILLME
+Right now, I develope on Ubuntu, I use vscode and geany.
 
 **Have you previously worked on a project of a similar scope?  If so, tell us
 about it.**
 
-FILLME
+I have worked on many projects, but not related to information retreval nor idexed search.
 
 **What timezone will you be in during the coding period?**
 
+
+
+GMT+2, Cairo,Egypt
+
+
 .. Please give at least the offset from GMT, but ideally also the timezone
 .. name so we aren't surprised by any differences around daylight savings
 .. time, which don't all line up in different parts of the world.
 
-FILLME
+
 
 **Will your Summer of Code project be the main focus of your time during the
 program?**
 
+Yes.
+
+
+
+
```

Comment:
This PR isn't as important as others, but it's a good idea to get into good habits. These additional lines shouldn't really be there. If you create commits by staging with confirmation (git add -p) then you'll see the diff hunks before you make the commits, and this will be easier to spot and avoid.

Reply:
Indeed - one of the reasons for us using restructured text in git and github PRs for proposal review is that these are tools that you'd need to use during your project (definitely for git and github PRs, and very likely for restructured text), so it helps you get a bit more familiar with these tools, and helps us see that you can use them appropriately.